### DISCUSSION TITLE
delete

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#699] [Feature Requests]
Genius is ten percent inspiration and fifty percent capital gains.

**Comment by nguyenkims** [comment]
The random alias is meant to be "one-click" creation so doesn't have any window to customise it. For choosing a different domain, you can use the custom alias right?

**Reply by nguyenkims** [reply]
Hi 1. is only true for Gmail as fas as I know as other email services use the dot sign (.) as a significant character. This risk is therefore quite small.

About 2., a simple way is to "invent" a random string like website_random@your-domain.com and uses it as alias.

I'm not sure to understand 3. though ... If the domain expires, you will loose all aliases associated to that domain.

> What I was hoping for was a similar page for a random alias to select the custom domain.

I wonder if the below option that sets the default domain for random aliases can be helpful in this case?

<img width="611" alt="image" src="https://user-images.githubusercontent.com/3167780/143215985-53700e38-0091-44a6-bc29-6621914cd3e4.png">

**Reply by nguyenkims** [reply]
> I have set that, but was just hoping for a way to make the process easier... like when click the Random Alias dropdown, to have an option to select the custom domain. I think I really wasn't understanding the New Custom Alias option though and how that could be used to do almost the same thing.

This can be a good idea but I'm afraid that the UI will become a bit too complex.
